src/index.ts

```diff
@@ -4,4 +4,11 @@ const app = new App({
   target: document.body,
 });
 
+declare global {
+  interface Window {
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    HASH_ROUTING: boolean;
```

Comment:
Any reason why we're not calling this `hashRouting`?

Reply:
This originated due to its first use as env variable.. But since we use it here as `Window` property, we can name it `hashRouting`

Reply:
Renamed with 79463006